functests/testlib.sh

```diff
@@ -99,10 +99,31 @@ wait_for_condition() {
 	local status=$6
 	for num in $( seq 1 ${max_tries} ); do
 		(( ${V} >= 1 )) && echo "waiting for $type condition availability: ${num}/${max_tries}"
-		if [ $(oc get ${kind} -o json ${NS} | jq '.items[0].status.conditions[] | select((.type=="'${type}'") and (.status="'${status}'"))' | wc -l) -gt 0 ]; then
+		if [ $(oc get ${kind} -o json ${NS} | jq 'select(.items[0].status.conditions != null) | .items[0].status.conditions[] | select((.type=="'${type}'") and (.status="'${status}'"))' | wc -l) -gt 0 ]; then
```

Comment:
could you please explain why do we need this change?

Reply:
When CR is newly created, it doesn't contain conditions array and due to that, jq complains:
`jq: error (at <stdin>:26): Cannot iterate over null (null)`

Reply:
What about using:

> Alternative operator: //
>
> A filter of the form a // b produces the same results as a, if a produces results other than false and null. Otherwise, a // b produces the same results as b.
> 
> This is useful for providing defaults: .foo // 1 will evaluate to 1 if there’s no .foo element in the input. It’s similar to how or is sometimes used in Python (jq’s or operator is reserved for strictly Boolean operations).